src/pal/attestation.cpp

```diff
@@ -279,49 +254,51 @@ namespace ccf::pal
 
     // chip_cert (VCEK) <-signs- sev_version (ASK)
     // ASK <-signs- root_certificate (ARK)
```

Comment:
```suggestion
    // ark_cert --signs--> ask_cert
    // ask_cert --signs--> vcek_cert
```

Lefts verb right more naturally, in my mind.